calin/nz/_posts/2022-06-26-overview.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: The Start of a Journey
+author: Călin Cașcaval
+---
+
+
+While saying goodbye to a number of friends on the latest hike I organized for our
+BARD hiking group, someone suggested that I post stories and photos of our new
+planned adventure in New Zealand. So I decided to start journaling -- something that
+I have been considering for a while, but I did not have a good incentive. While we
+have been spending time outdoors, hiking, biking, and skiing, and I have posted a
+number of my pictures either on
+[500.px](https://500px.com/CalinCascaval){:target="_blank"} or shared privately,
+writing text about those has been not something I considered. After all, we want
+these hikes and skiing trips to be uneventful! So here we are, at the beginning of a
+new journey, one that feels exciting and gives me a good reason to challenge myself
+to write.
+
+Of course, everyone is asking why are we moving? The answer is ... complicated: it's
+been a while that we have been considering moving out of the US. We had an amazing
+run of 29 years here, and we are grateful for what this country has offered us:
+
+- **free education**, and actually even better -- we were paid to go to school,   learn, and earn our degrees.
+- **family life**: the opportunity to build a family; we have two loving kids who continue to teach and inspire us.
+- **job opportunities** in industries that have been growing exponentially and allowed us to live the "American dream": work hard and have a comfortable life.
+- **acceptance**: we never felt excluded or oppressed, even though we have  _funny accents_. It used to be that folks used that opportunity to ask   where are we from. It is also true that we _fit in_ -- smack in the middle of the white, middle class, educated class.
```

Comment:
Fit in not only with existing Americans, but also with the thriving community of immigrants and people with "funny accents" who make up the majority of your friends and colleagues (and whose kids continue to make up the majority of mine), all of whom call this country home now

(also: we are well, well, beyond middle class; even if socially you are much more modest than the people living it large outside their means. it's hard to categorize immigrants in technical fields, because the incomes are very high but the ego is not, which doesn't map to our existing notions of what a "class" is. not the post to go into all that, but just a thought.)

Reply:
I still consider ourselves be middle class -- with means, but not rich, even though by some standards we're considered rich. You kind of put your finger on it, we're hard to categorize. And moreover, while we adapted well to some of the American ways of life, we kept a whole lot of values that are at odds with their style.

That said, I did adopt your suggestion of fitting in with emigrants.